pa_matchbatch.sh

```diff
@@ -145,7 +146,7 @@ getJiraFixedInFor() {
 		;;
 		marionette-collective) echo MCO
 		;;
-		puppet-resource_api) echo PDK
+		puppet-resource_api) echo RSAPI
```

Comment:
The rsapi actually uses the `PDK` project but an `RSAPI <version>` fixVersion -- apparently fixVersions are not diligently applied, but it sounds like this is most correct